10_Examples/jett-onboarding/package.xml

```diff
@@ -0,0 +1,21 @@
+<?xml version="1.0"?>
+<?xml-model href="http://download.ros.org/schema/package_format3.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
+<package format="3">
+  <name>jett-onboarding</name>
+  <version>0.0.0</version>
+  <description>TODO: Package description</description>
+  <maintainer email="jettvanbrocklin@todo.todo">jettvanbrocklin</maintainer>
+  <license>TODO: License declaration</license>
+
+  <buildtool_depend>ament_cmake</buildtool_depend>
+
+  <depend>rclcpp</depend>
+  <depend>std_msgs</depend>
+
```

Comment:
So, calling that `find_package` up in the CMake is the primary way that we list dependencies and we would link to libraries in a different package, but we list them here.

The main reason for listing them here, is if you have packages that depend on each other within the same ROS workspace (so for VEXU, thats just our whole repo), listing dependency tells colcon what order to build things in.

So, if pkg A lists pkg B as a dependency here, then when you build the full repo, it wont start compiling A until B is totally done. Otherwise A will look for stuff that doesnt exist yet, and fail.